### DISCUSSION TITLE
major release is listed in PR

### DISCUSSION THREAD

**Discussion by codewode** [renovatebot/renovate#13687] [Request Help] [answered]
how do I restrict renovate to create PR only for minor or patch version, i have the following config, but it still creates PR having dependency with "major" updates.

```
{
  "platform":"azure",
  "rangeStrategy":"auto",
  "baseBranches":[
    "main"
  ],
  "labels":[
    "renovate"
  ],
  "groupName": "all dependencies",
  "separateMajorMinor": false,
  "groupSlug": "all",
  "packageRules":[
    {
      "matchUpdateTypes":[
        "minor",
        "patch"
      ],
      "matchPackagePatterns": [
        "*"
      ],
      "groupName": "all dependencies",
      "groupSlug": "all"
    }
  ]
}
```

**Comment by viceice** [comment, marked as answer]
`"separateMajorMinor": false,` is causing that, remove it, and also global `groupName` and `groupSlug`

**Reply by codewode** [reply]
that did not help out, unfortunately. So I understand the behavior that removing ```"separateMajorMinor": false``` will still create a separate PR with "major". And I have observed it too after running the pipeline.

the intention to have global ```groupName``` and ```groupSlug``` was to have a single PR, instead of several PR's.

is it the intended behavior that ```major``` upgrades from the package files cannot be explicitly turned off?

**Reply by rarkins** [reply]
We don't recommend grouping major updates unless they're related packages.

**Reply by codewode** [reply]
as suggested I have removed ```groupName``` and ```groupSlug```, also ```"separateMajorMinor": false``` is removed. Now, I see 3 PR's created with the following titles

1. chore(deps): update all dependencies (major)
2. chore(deps): update all dependencies
3. chore(deps): pin dependencies

What I wanted to achieve is to completely get rid of ```major``` updates. Because it requires me to install the framework.Net6, whereas I am still on.Net5 and due to some restricted policies upgrading to a new framework is not yet allowed. So the PR with the title ```major``` is useless for me and I would have to independently mark them as abandoned in almost 10 different repos.

```
{
  "platform":"azure",
  "rangeStrategy":"auto",
  "baseBranches":[
    "main"
  ],
  "labels":[
    "renovate"
  ],
  "packageRules":[
    {
      "matchUpdateTypes":[
        "minor",
        "patch"
      ],
      "matchPackagePatterns": [
        "*"
      ],
      "groupName": "all dependencies",
      "groupSlug": "all"
    }
  ]
}
```

**Reply by rarkins** [reply]
Add a package rule with matchUpdateTypes=major and enabled=false

**Reply by codewode** [reply]
thanks, it works!
